stages/org.osbuild.chown

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+"""
+Change file owner and group
+Change the file user and/or group ownership of one or more files or directories inside the tree.
+"""
+
+import os
+import subprocess
+import sys
+
+import osbuild.api
+from osbuild.util.path import in_tree
+
+SCHEMA = r"""
+"additionalProperties": false,
+"required": ["paths"],
```

Comment:
I winder whether we should not require that at least one of `user` or `group` is specified, because otherwise this is noop, which could cover potential bugs. IOW it does not make sense to use the stage in such case 🤔